src/displayapp/screens/Notifications.cpp

```diff
@@ -79,6 +79,12 @@ void Notifications::Refresh() {
       timeoutLinePoints[1].x = pos;
       lv_line_set_points(timeoutLine, timeoutLinePoints, 2);
     }
+
+    if(!this->isTitleScrolling && tick >= timeoutTickCountStart + timeoutStartTitleScrolling){
+    	currentItem->StartTitleScroll();
+        this->isTitleScrolling = true;
```

Comment:
The structure is not clear to me. Line 84 is hidden behind a function call, but the bool is set outside of that function. I would dismiss the function and just run the `lv_...` from line 369 here directly.

Reply:
That was the first that I did, but I realized that I needed to put `alert_type` as a public member of `notificationItem` so I thought it was better to use a function. The boolean prevents multiple calls to the same function.
As far as I understand, though, you'd prefer the first alternative, am I right?

Reply:
I didn't notice that they are different objects, sorry. That makes more sense. Then I have a new question: Do you think it would be good to reset the Boolean when `currentItem` gets changed, like in line 39?

Reply:
Yeah, you are right... I didn't test a second long notification. Actually, we should check what happens when a new notification arrives (because I already forgot 🤣) so yeah, I'll make a commit in the near future to solve this. Thank you! :)

Reply:
Good catch! I think there is a separate notification preview screen that you need to change too.